atomic_reactor/plugins/build_source_container.py

```diff
@@ -27,7 +27,10 @@ class SourceContainerPlugin(BuildStepPlugin):
     key = PLUGIN_SOURCE_CONTAINER_KEY
 
     def export_image(self, image_output_dir):
-        output_path = os.path.join(tempfile.mkdtemp(), EXPORTED_SQUASHED_IMAGE_NAME)
+        os.makedirs(os.path.join(self.workflow.source.workdir, 'temp'), exist_ok=True)
+        output_path = os.path.join(tempfile.mkdtemp(dir=os.path.join(self.workflow.source.workdir,
+                                                                     'temp')),
+                                   EXPORTED_SQUASHED_IMAGE_NAME)
```

Comment:
I wouldn't make this a tmpdir, let's just use something like `{workdir}/{EXPORTED_SQUASHED_IMAGE_NAME}`. The output is not temporary, it is the most important output in the source container pipeline (the built image)

Reply:
updated.